web/controllers/account_controller.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Notox.AccountController do
+  use Notox.Web, :controller
+  alias Notox.User
+
+  def show(conn, %{"id" => id}) do
+    user = Repo.get(User, id)
```

Comment:
There's already a controller/action that does this... the user controller. Is this any different than that? Should it only show the current user?